documentation/modules/post/windows/gather/credentials/solarwinds_orion_dump.md

```diff
@@ -0,0 +1,263 @@
+This module exports and decrypts credentials from SolarWindows Orion Network Performance Monitor
```

Comment:
I did not even know this was a thing, will do. This will be cleaned up in the next commit, and I will be sure to invoke the same before a PR in the future. I  get `[INFO] Missing Section: ## Options` as the only finding now, but assume that is permissible.